contrib/examples/create-cluster.sh

```diff
@@ -20,12 +20,14 @@ oc get secret aws-credentials -n cluster-operator -o yaml | sed -E '/(namespace:
 oc get secret ssh-private-key -n cluster-operator -o yaml | sed -E '/(namespace:|annotations|last-applied-configuration:|selfLink|uid:|resourceVersion:)/d' | oc apply -f -
 oc get secret ssl-cert -n cluster-operator -o yaml | sed -E '/(namespace:|annotations|last-applied-configuration:|selfLink|uid:|resourceVersion:)/d' | oc apply -f -
 
+default_name="$(whoami)-cluster"
```

Comment:
I would rather not check for ${whoam}-cluster.yaml. I think it makes the logic of which yaml file is used a bit convoluted where someone might unknowingly use the wrong file.

@joelddiaz How would you feel if you had to specify the name of the yaml file that you use instead of relying on a default ${whoami}-cluster.yaml?

Reply:
@staebler fine by me to have the user specify the yaml file